modules/ROOT/pages/clauses/create.adoc

```diff
@@ -23,7 +23,16 @@ Multiple labels are separated by colons.
 CREATE (charlie:Person:Actor {name: 'Charlie Sheen'}), (oliver:Person:Director {name: 'Oliver Stone'})
 ----
 
-Creates two nodes, bound to the variables `charlie` and `oliver`, each with a `Person` label and a `name` property.
+Multiple labels can also be separated by an ampersand `&`, in the same manner as it is used in xref:patterns/reference.adoc#label-expressions[label expressions].
+Separation by colon `:` and ampersand `&` cannot be mixed in the same query.
```

Comment:
is it the same query or the same clause?

for example, would this be allowed:

```
CREATE (charlie:Person&Actor {name: 'Charlie Sheen'}),
MATCH (charlie)
CREATE (charlie)-[:LOVES]->(oliver:Person:Director {name: 'Oliver Stone'})
```

(ampersand in the first CREATE clause, colon in the second CREATE clause)

Reply:
Good catch! It is actually per clause and not query